cpu/cpu_linux.go

```diff
@@ -4,12 +4,14 @@ import (
 	"bufio"
 	"os"
 	"regexp"
-	"strconv"
+	// "strconv"
 )
 
 var cpuMap = map[string]string{
 	"vendor_id":  "vendor_id",
 	"model name": "model_name",
+	"cpu cores":  "cpu_cores",
+	"siblings":  "cpu_logical_processors",
```

Comment:
Tiny nitpick: adding a space to align `"cpu_logical_processors"` with the rest :p